app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardActivityTest.kt

```diff
@@ -134,6 +134,7 @@ class RevisionCardActivityTest {
   lateinit var monitorFactory: DataProviderTestMonitor.Factory
 
   private val profileId = ProfileId.newBuilder().apply { internalId = 1 }.build()
+  private val fractionsSubtopicListSize = 4
```

Comment:
Suggest making a constant at the top for the test (in the same way as RevisionCardFragmentTest).

Reply:
done